tests/devices/conftest.py

```diff
@@ -32,3 +38,147 @@ def create_temporary_toml_file(request) -> str:
             f.write(dedent(content))
         request.node.toml_file = toml_file
         yield
+
+
+@pytest.fixture(scope="session")
+def create_operation_from_name():
```

Comment:
what is the purpose of this?

Reply:
Marked as resolved since previous unfinished tests have been discarded therefore comments here become outdated